frontend/src/pages/PickupGem/MessageDisplay.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="bg-container" :style="addBackground">
+    <BottomDrawer :gemPackage="gemPackage"/>
+  </div>
+</template>
+<script  lang="ts">
+import { defineComponent, PropType } from "vue";
+import { GemInfo } from "@/interfaces";
+import BottomDrawer from "@/components/MessageBottomSheet.vue";
```

Comment:
Is it intentional that BottomDrawer and MessageBottomSheet have different names?

Reply:
Nobody could decide what the component is called online... then I forgot to come to a conclusion D: Lemme standardise!